_developer-files/connect/api/objects/form-properties/sources/saas/recharge-object.md

```diff
@@ -0,0 +1,54 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/connect-templates/destination-form-property/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#        CONTENT TYPE        #
+# -------------------------- #
+
+product-type: "connect"
+content-type: "api-form"
+form-type: "source"
+key: "source-form-properties-recharge-object"
+
+
+# -------------------------- #
+#        OBJECT INFO         #
+# -------------------------- #
+
+title: "ReCharge Source Form Property"
+api-type: "platform.recharge"
+display-name: "ReCharge"
+
+source-type: "saas"
+docs-name: "recharge" # This should be whatever integration.name is. Ex: LinkedIn Ads is linkedin-ads
+
+property-description: ""
```

Comment:
Don't forget to remove this if it's not being used - it can cause display issues because of the way Liquid checks for it.

So, in Liquid, you can check for the presence of a variable by using this:

```liquid
{% if form-property.property-description %}
...
{% endif %}
```

This basically says "Hey, if this thing is merely present, even if it's empty (aka an empty string), do this stuff." [Liquid docs on this are here](https://help.shopify.com/en/themes/liquid/basics/true-and-false#truthy).

The logic in our templates that looks at this has a check like this - if `property-description` is present, even if it's empty, it'll change the way the description for a source form property is displayed. Most form properties have descriptions along the lines of the following:

```
[SOURCE] connections read data from the [SOURCE] API and correspond to source type: platform.[name].
```

But [here's an example of where `property-description` is used](https://www.stitchdata.com/docs/developers/stitch-connect/api#source-form-properties-heap-object) instead:

```
Heap connections read data from data dumps in Amazon S3 buckets and correspond to source type: platform.heap.
```

I think the template for this can be updated to include a reminder, or to comment it out by default, but I just wanted to explain how this works. :) We can take a closer look at the API templates together if you'd like to know more about how the Liquid works.